components/camel-twitter/src/main/docs/twitter-search-component.adoc

```diff
@@ -31,6 +31,15 @@ The Twitter Search component supports 10 options, which are listed below.
 
 ### Endpoint Options
 
+The search keywords query should be specified accordingly to https://developer.twitter.com/en/docs/tweets/search/guides/standard-operators[twitter search operators].
+
+Examples of a search query:
+
+* `syndesis OR fuse`
```

Comment:
Can we provide some other examples that are dogs and cats like, eg not referring to a commercial vendors product name

Reply:
Done.